src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -106,21 +121,21 @@ class AnnualIncomeSection extends React.Component {
         <div className="input-section">
           <h6>Spouse</h6>
           <ErrorableTextInput
-              errorMessage={getErrorMessage(this.props.data.spouseGrossIncome, message)}
+              errorMessage={getErrorMessage(this.props.data.spouseGrossIncome)}
```

Comment:
The spouse and child income is going to have to be required as well if they have a spouse or dependents.